plextraktsync/plex_api.py

```diff
@@ -227,7 +227,8 @@ def collected_at(self):
 
     @property
     def parts(self) -> List[MediaPart]:
-        for media in self.item.media:
+        item = self.plex.fetch_item(self)
```

Comment:
Should this be `self.item.key`? 

Without grabbing the key I receive the below error:

```requests.exceptions.InvalidURL: Failed to parse:  https://<IP>.<ID>.plex.direct:32400<tmdb:509967:<Movie:129024:6-Underground>>```

Reply:
`self.plex` is also `None` when invoked via inspect. Not sure where that should be fixed, but locally was able to fix it by passing `plex=self` at:

- https://github.com/Taxel/PlexTraktSync/blob/51f22c1abffb886017b076887798a9d6c22ca529/plextraktsync/plex_api.py#L504

Reply:
It looks like #659 would fix the `None` issue?

Reply:
1. 2bca7ef3. 
2. `None` problem? Do you have an error?

Reply:
Can't repo at the moment as I'm running a full sync so it'll be about 40 minutes, but you should be able to reproduce by running `plextraktsync inspect <ID>`

Reply:
Indeed, https://github.com/Taxel/PlexTraktSync/pull/659 is needed here

Reply:
Rebased after https://github.com/Taxel/PlexTraktSync/pull/659  merge